packages/documentation-site/package.json

```diff
@@ -20,9 +20,9 @@
     "@patternfly/chatbot": "2.2.0-prerelease.37",
     "@patternfly/design-tokens": "1.13.2",
     "@patternfly/documentation-framework": "workspace:*",
-    "@patternfly/quickstarts": "6.2.0-prerelease.7",
+    "@patternfly/quickstarts": "6.2.0-prerelease.8",
     "@patternfly/react-catalog-view-extension": "6.1.0-prerelease.4",
-    "@patternfly/react-component-groups": "6.2.0-prerelease.7",
+    "@patternfly/react-component-groups": "6.2.0-prerelease.8",
     "@patternfly/react-console": "6.0.0",
     "@patternfly/react-data-view": "6.1.0",
     "@patternfly/react-docs": "7.2.0-prerelease.54",
```

Comment:
I think this should be updated to [prerelease.58](https://www.npmjs.com/package/@patternfly/react-docs?activeTab=versions)